index.bs

```diff
@@ -1719,6 +1719,45 @@ that should be considered in the development of new features, notably:
 * [[#feature-detect]]
 * Polyfill development should be encouraged
 
+<h3>Extend existing manifest files rather than creating new ones</h3>
+
+Manifest files are sometimes required. However, they can also add complexity to the web platform,
+which creates developer confusion. The use of different manifests on the web should therefore be kept to a minimum.
+
+Metadata can reside inside a new manifest if that is related to a different function or the fetch timing 
+of that metadata is different. Application metadata should be added to the Web App Manifest or be an extension of it. 
+Parameters for a specific function (like Payment Method Manifest) are a completely different type. Publication Manifest,
+using JSON-LD is another type as well.
+
+Note that in all cases, the naming conventions should be harmonized (see section 2.2)
+ 
+[Note: Due to historical reasons, existing manifests use lowercase, underscore-delimited names. There have been times 
+where it has been beneficial to re-use dictionaries from a manifest in DOM APIs as well, which are camel-cased. 
+One such example is the <a href="https://w3c.github.io/image-resource/">image resource</a>. For this reason, if a 
+key can clearly be expressed as a single word, that is recommended.]
+ 
+When designing new keys and values for a manifest, make sure they are really needed (thus, define well thought out use-cases).
+Also please check if a similar key exists. If an existing key/value pair does more or less what is needed, 
+work with the existing spec to extend it to your use-case if possible.
+
+Some of the existing manifest files include
+
+* <a href="https://www.w3.org/TR/appmanifest/">Web App Manifest</a>
+* <a href="https://www.w3.org/TR/payment-method-manifest/">Payment Method Manifest</a>
+* <a href="https://www.w3.org/TR/pub-manifest/">Publication Manifest</a>
```

Comment:
You have plenty of examples already, but I'll offer origin policy as another: https://wicg.github.io/origin-policy/. In particular that one exemplifies the different fetch timing and scope (see https://github.com/WICG/origin-policy/blob/master/policy-format.md#appendix-relation-to-web-app-manifest).

Reply:
Added this example.